sagemaker-featurestore/feature_store_kms_key_encryption.ipynb

```diff
@@ -0,0 +1,554 @@
+{
```

Comment:
<p>What is <code>SageMakerFeatureStore-</code>,  and why it showed up as a grant? I don't see you ever called  <code>kms:CreateGrant</code>  so how did this grant show up?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
These grants are created by FeatureStore service, we impersonate customers by using their FAS credentials then we create the grants for our service roles so that we are gonna be able to use customer provided KmsKey to encrypt their online store. So after feature group creation is done, customers should be able to see a couple of more grants created by us.

Reply:
I see. This description of where the grants come from should be included in the notebook.

Reply:
Added more context to clarity this.